cmd/describe/cluster/cmd.go

```diff
@@ -703,6 +713,7 @@ func formatClusterHypershift(cluster *cmv1.Cluster,
 		upgrade["nextRun"] = scheduledUpgrade.NextRun().Format("2006-01-02 15:04 MST")
 		ret["scheduledUpgrade"] = upgrade
 	}
+	ret["displayName"] = displayName
```

Comment:
This (and L687 above) are for --output=json and --output=yaml, right?

That output contains tons of snake_case field names, and very few lowerCamelCase names:
```yaml
...
delete_protection:
  enabled: false
disable_user_workload_monitoring: true
displayName: dt-hcp-test-4  ⬅️
dns:
  base_domain: jw4b.p3.openshiftapps.com
etcd_encryption: false
...
```
Is it desirable to mix naming conventions, and to rename a field that already exists in API as `display_name`? :thinking:  Or is that idea to set apart all fields that are not part of _same_ API?

your call, just asking.  LGTM either way

Reply:
moved to snake_case, good call